lib/auth/auth.go

```diff
@@ -4200,13 +4207,48 @@ func (a *Server) UpsertDatabaseServer(ctx context.Context, server types.Database
 	return lease, nil
 }
 
+// UpdateOSSDesktopAlert manages alert to show OSS users if there are more than 3 non-AD desktops
+func (a *Server) UpdateOSSDesktopAlert(ctx context.Context) error {
+	desktops, err := a.GetWindowsDesktops(ctx, types.WindowsDesktopFilter{OnlyNonAD: true})
```

Comment:
Should we do the build type check first? There's no reason to fetch all the desktops if we're on enterprise, right?

@rosstimothy is there a more efficient way to just get the count without requiring us to unmarshal a bunch of resources from the backend?